package.json

```diff
@@ -28,14 +28,18 @@
     "build": "react-scripts build",
     "server": "nodemon server/server.js",
     "test": "react-scripts test --env=jsdom",
+    "test2": "jest --coverage",
```

Comment:
Is there any reason why you need test and test2 if you're not using the built-in testing from create react app?